lib/prefix.c

```diff
@@ -1124,6 +1124,19 @@ const char *prefix2str(union prefixconstptr pu, char *str, int size)
 	return str;
 }
 
+void prefix_mcast_ip_dump(const char *onfail, const struct ipaddr *addr,
+			  char *buf, int buf_size)
+{
+	int save_errno = errno;
```

Comment:
Why do we need to save the errno?  What is the failure case here that needs this?  We don't typically need to do this anywhere else in the code.

Reply:
That was done in case the caller needed it. I've removed this.